api/loader/fileloader_test.go

```diff
@@ -14,6 +14,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
```

Comment:
We can fix this in your other PR

Ref: https://github.com/kubernetes-sigs/kustomize/pull/4652#discussion_r887377378